cilium/cmd/bgp_peer_get.go

```diff
@@ -46,11 +46,12 @@ func printSummary(peers []*models.BgpPeer) {
 		return peers[i].LocalAsn < peers[j].LocalAsn || peers[i].PeerAddress < peers[j].PeerAddress
 	})
 
-	fmt.Fprintln(w, "Local AS\tPeer AS\tPeer Address\tSession\tUptime\tFamily\tReceived\tAdvertised")
+	fmt.Fprintln(w, "Local AS\tPeer AS\tPeer Address\tPeer Address\tSession\tUptime\tFamily\tReceived\tAdvertised")
```

Comment:
nit : Address -> Port, although I do not think we need port information in summary output.
  
Json output contains more detailed information of peer, it will have peer-port field.

Reply:
Or we can display it like `10.0.0.1:179". What do you think @harsimran-pabla?

Reply:
JSON fields should be separated of course.

Reply:
Yes, '10.0.0.1:179' output is great.